src/components/NewsRoomCard.jsx

```diff
@@ -25,7 +25,7 @@ const NewsRoomCard = props => {
           <div className="col dg_news_date-author">
             <p>
               <span>{published}</span>
-              <span>{author}</span>
+              <span>{author ? author : "Baltimore County"}</span>
```

Comment:
in a case like this you can simplify to:

```js
author || "Baltimore County"
```

another option in this case would be to set this as the default value when destructring on line 11

```js
const { publishDate, title, author = "Baltimore County", articleSummary, url } = props;
```

either is fine.

Reply:
done